### PR TITLE
celinanperalta/oap 13

### PR DESCRIPTION
**Pull request by celinanperalta**
- Change python version to 3.10
- Add config and connection for database (local testing only)
- Add sklearn, pandas to pipfile
- Update gitignore

### CODE REVIEW

**Comment by celinanperalta** [comment]
Note: this doesn't really close oap-13: need to work with @max-zaremba to design db schema

File: oapen-engine/src/data/config.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+from configparser import ConfigParser
+
+def config(filename='../database.ini', section='postgresql'):
```
Comment (justinoboyle) [inline oapen-engine/src/data/config.py:4]: maybe add to `README.md` if `database.ini` needs to be created by user?

**Review by justinoboyle — APPROVED** [review]

**Comment by eshellman** [comment]
have you shared where OAP-* comes from?

**Comment by celinanperalta** [comment]
Working on Linear integration w/ Github, will send invite to project board

**Pull request merged**